frontend/src/components/Spinner/index.tsx

```diff
@@ -1,13 +1,41 @@
-import { LoadingOutlined } from '@ant-design/icons';
```

Comment:
I see more occurrence of `<LoadingOutlined />` in code base, so do we want to change those instances also. Can you please check if this request holds true there also

Reply:
Done :slightly_smiling_face: